test/spec/modules/userId_spec.js

```diff
@@ -786,6 +791,52 @@ describe('User ID', function() {
       }, {adUnits});
     });
 
+    it('test hook from sharedId html5', function(done) {
+      // simulate existing browser local storage values
+      localStorage.setItem('sharedId', JSON.stringify({'sharedId': 'test_sharedId'}));
```

Comment:
I'm not 100% sure where in this line (and elsewhere in this file) `sharedId` would need to be changed to `sharedid`